Specifications/Events/for_EventStore/when_committing_an_uncommitted_event.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Dolittle.SDK.Events.for_EventConverter.given;
+using Machine.Specifications;
+using It = Machine.Specifications.It;
+
+namespace Dolittle.SDK.Events.for_EventStore
+{
+    public class when_committing_an_uncommitted_event : given.an_event_store_and_an_execution_context
+    {
+        static UncommittedEvent uncommitted_event;
+        static an_event content;
+        static EventSourceId event_source;
+        static EventType event_type;
+        static bool is_public;
+
+        Establish context = () =>
+        {
+            content = new an_event("goodbye world", 12345, false);
+            event_source = new EventSourceId(Guid.NewGuid());
+            event_type = new EventType(Guid.NewGuid());
+            is_public = false;
+            uncommitted_event = new UncommittedEvent(event_source, event_type, content, is_public);
+        };
+
+        Because of = async () => await event_store.Commit(uncommitted_event);
+        It should_convert_the_object_to_uncommitted_events = () => events_to_convert.ShouldBeOfExactType<UncommittedEvents>();
```

Comment:
It should be enough to check that it tries to convert the specific `uncommitted_event`. So something like:
```csharp
It should_convert_the_provided_event = () => events_to_convert.ShouldContainOnly(uncommitted_event);
```